pkg/cmd/server/start/start_master.go

```diff
@@ -307,6 +315,9 @@ func (o MasterOptions) CreateCerts() error {
 func StartMaster(openshiftMasterConfig *configapi.MasterConfig) error {
 	glog.Infof("Starting master on %s (%s)", openshiftMasterConfig.ServingInfo.BindAddress, version.Get().String())
 	glog.Infof("Public master address is %s", openshiftMasterConfig.AssetConfig.MasterPublicURL)
+	if len(openshiftMasterConfig.DisabledFeatures) > 0 {
+		glog.Infof("Disabled features: %s", strings.Join(openshiftMasterConfig.DisabledFeatures, ", "))
```

Comment:
`glog.V(4).Infof`.  We don't want want to advertise to the admin that we're disabling his features.